week-5/index.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+
+const PersonService = require('./services/person-service')
+
+const app = express()
+
+app.set('view engine', 'pug')
+app.use(bodyParser.json())
+
+app.get('/', (req, res) => {
+  res.render('index')
+})
+
+app.get('/person/all', async (req, res) => {
+  const people = await PersonService.findAll()
+  res.render('person', { people })
+})
+
+app.get('/person/:id', async (req, res) => {
+  const user = await PersonService.find(req.params.id)
+  console.log({user})
```

Comment:
should we remove the `console.log` here?

Reply:
Yes, sure. We don't need it.